### DISCUSSION TITLE
Domain move / alias issue

### DISCUSSION THREAD

**Discussion by SilentCaesar** [vercel/vercel#5126] [General]
Hi all,

I'm having trouble moving a domain from a hobby plan over to a team plan. Once I delete all `alias`ed domains attached, and attempt to move it, it still says there are alias' remaining.

Can I get some support for this, no one seems to be answering.

Thank you.

**Comment by mcsdevv** [comment]
Hello @SilentCaesar, you can list the outstanding alias uses with Vercel CLI and `vercel alias ls`.

**Reply by SilentCaesar** [reply]
@mcsdevv is there a private channel or support email I can reach, I tried support@vercel to no luck.

**Reply by mcsdevv** [reply]
You can submit tickets using the support form available from the dashboard, just click 'Support'.
